# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/external-memory/src/handle.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/external-memory/src/handle.rs b/third_party/rust/external-memory/src/handle.rs
deleted file mode 100644
--- a/third_party/rust/external-memory/src/handle.rs
+++ /dev/null
@@ -1,20 +0,0 @@
-#[derive(Debug)]
-/// Windows handle.
-pub struct Handle(*mut std::ffi::c_void);
-impl From<*mut std::ffi::c_void> for Handle {
-    fn from(ptr: *mut std::ffi::c_void) -> Self {
-        Self(ptr)
-    }
-}
-#[cfg(windows)]
-impl std::os::windows::io::AsRawHandle for Handle {
-    fn as_raw_handle(&self) -> std::os::windows::raw::HANDLE {
-        self.0
-    }
-}
-impl std::ops::Deref for Handle {
-    type Target = *mut std::ffi::c_void;
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}